tests/SprykerEcoTest/Zed/Payone/_support/Business/DataProvider/PayoneDistributePricesDataProvider.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\Payone\Business\DataProvider;
+
+use Generated\Shared\Transfer\ExpenseTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\TotalsTransfer;
+
+trait PayoneDistributePricesDataProvider
+{
+    /**
+     * @return array
+     */
+    public function provideOrdersData(): array
+    {
+        return [
+            $this->provideOrdersDataWithoutExpenses(),
+            $this->provideOrdersDataWithoutItems(),
+            $this->provideOrdersDataWithItemsAndExpenses(),
+            $this->provideOrdersDataWithOnlyPayonePayment(),
+        ];
+    }
+
+    /**
+     * @return array
+     */
+    protected function provideOrdersDataWithoutExpenses()
+    {
+        return [
+            [
+                [
```

Comment:
You can create the transfer objects in the data provider.

Reply:
Fixed